src/main/java/seam/Image.java

```diff
@@ -0,0 +1,98 @@
+package seam;
+
+import javax.imageio.ImageIO;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.InputStream;
+
+public class Image
+{
+    private int imageWidth;
+    private int imageHeight;
+    private Pixel[][] pixels;
+    private EnergyCalculator energyCalculator;
+
+    public Image(String imagePath)
+    {
+        try (InputStream seamImage = Image.class.getResourceAsStream(imagePath))
+        {
+            BufferedImage image = ImageIO.read(seamImage);
+
+            imageWidth = image.getWidth();
+            imageHeight = image.getHeight();
+            pixels = new Pixel[imageWidth][imageHeight];
+            for (int x = 0; x < imageWidth; x++)
+            {
+                for (int y = 0; y < imageHeight; y++)
+                {
+                    pixels[x][y] = new Pixel(image.getRGB(x, y));
+                }
+            }
+
+            energyCalculator = new EnergyCalculator(pixels, imageWidth, imageHeight);
+            energyCalculator.calculateEnergy();
+            energyCalculator.calculateVerticalEnergy();
+            energyCalculator.calculateHorizontalEnergy();
+        } catch (Exception exception)
+        {
+            exception.printStackTrace();
```

Comment:
In this situation, if an Exception is ever thrown here you want the program to exit. You don't just want to print the stack trace.
So here it would be more correct to declare this method `throws Exception`. Also it's never a great idea to catch Exception, you probably want to only catch IOException here.